### PR TITLE
feat: Add setting up fdw

### PR DESCRIPTION
**Pull request by xieydd**

### CODE REVIEW

File: .vitepress/config.mts [hunk 1]
```diff
@@ -89,6 +89,7 @@ export default defineConfig({
             { text: 'Configuration', link: '/admin/configuration' },
             { text: 'Upgrading from older versions', link: '/admin/upgrading' },
             { text: 'Kubernetes', link: '/admin/kubernetes' },
+            { text: 'Setting up FDW', link: '/admin/fdw' },
```
Comment (gaocegege) [inline .vitepress/config.mts:92]: Could we use this?
```suggestion
            { text: 'Foreign Data Wrapper (FDW)', link: '/admin/fdw' },
```

File: src/admin/fdw.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
+# Setting up FDW reduce the pressure on the primary instance
+
+Vector retrieval is a query that consumes CPU and IO, even if there already have an index. If this query is run on the primary instance, it can negatively impact its performance. To alleviate this issue, it is recommended to execute the query on a specific PostgreSQL cluster, which will reduce the pressure on the primary instance.
+
+Fortunately, PostgreSQL provides the [`postgres_fdw`](https://www.postgresql.org/docs/current/postgres-fdw.html) module, which includes the foreign-data wrapper. This wrapper can be used to access data stored in external PostgreSQL servers. This article will explain how to use `postgres_fdw` to access index data in a foreign PostgreSQL cluster that already has the [`pgvecto.rs`](https://github.com/tensorchord/pgvecto.rs) extension installed.
```
Comment (gaocegege) [inline src/admin/fdw.md:5]: Could we explain what FDW is here?

File: .vitepress/config.mts (hunk 1 above)
Comment (xieydd) [reply .vitepress/config.mts:92]: lgtm

File: src/admin/fdw.md (hunk 2 above)
Comment (xieydd) [reply src/admin/fdw.md:5]: yep, I will add this part.

File: src/admin/fdw.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
+# Setting up Foreign Data Wrapper (FDW) reduce the pressure on the primary instance
+
+Vector retrieval is a query that consumes CPU and IO, even if there already have an index. If this query is run on the primary instance, it can negatively impact its performance. To alleviate this issue, it is recommended to execute the query on a specific PostgreSQL cluster, which will reduce the pressure on the primary instance.
+
+[Foreign data wrapper(FDW)](https://wiki.postgresql.org/wiki/Foreign_data_wrappers) is a library that you can use to access and interact with an external data (foreign data) source. They allow you to query foreign objects from remote servers as if they were local objects. Postgres now has a lot of foreign data wrappers available and they work with plenty of different source types: NoSQL databases, platforms like Twitter and Facebook, geospatial data formats, etc.
```
Comment (VoVAllen) [inline src/admin/fdw.md:5]:
```suggestion
[Foreign data wrapper(FDW)](https://wiki.postgresql.org/wiki/Foreign_data_wrappers) is a module that you can use to access and interact with an external data (foreign data) source. They allow you to query foreign objects from remote servers as if they were local objects. Postgres now has a lot of foreign data wrappers available and they work with plenty of different source types: NoSQL databases, platforms like Twitter and Facebook, geospatial data formats, etc. 
```

**Review by VoVAllen — APPROVED** [review]
LGTM

**Pull request merged**